_rules/SC4-1-1-unique-id.md

```diff
@@ -20,13 +20,13 @@ authors:
 
 ### Applicability
 
-Any element that has an `id` attribute.
+Any `id` attribute specified on an HTML or SVG element.
 
-**Note:** Elements that are neither [exposed to assistive technologies](#exposed-to-assistive-technologies) nor [visible on the page](#visible-on-the-page) are still considered for this rule.
+Elements that are neither [exposed to assistive technologies](#exposed-to-assistive-technologies) nor [visible on the page](#visible-on-the-page) are still considered for this rule.
```

Comment:
Why was the "Note" identifier removed? Seems appropriate to keep this as a note because it does not define the applicability (it only clarifies the meaning).

Reply:
Because you said so (https://github.com/auto-wcag/auto-wcag/pull/285#discussion_r223695439), and I always trust you :)
Now you can decide if I need to trust past-Shadi or present-Shadi ;)

Reply:
hahahaha - never trust any-shadi!!!

I looked up my past comment. It looked to me like a requirement because it linked to nifty definitions. But reading it more closely, I think it is a clarification and not a requirement. I'm sorry for the confusion.

Reply:
**Note:** identifier is back again.